copier/user_data.py

```diff
@@ -359,6 +367,11 @@ def get_multiline(self) -> bool:
 
     def validate_answer(self, answer) -> bool:
         """Validate user answer."""
+        if self.regex is not None:
+            return (
+                self._re.match(answer) is not None
```

Comment:
I think a search would allow for less strict regex, while still allowing exact matches with `^` and `$` at the boundaries of the regex.

Reply:
Thanks. I'll take a look while writing the tests.